Home/styles.css

```diff
@@ -111,4 +112,17 @@ img {
 	max-width: 100%;
 }
 
+footer {
+	margin-top: 80px;
+	margin-left: 10px;
+	background-color: rgb(82, 80, 80);
```

Comment:
Is this one of the colors we chose for the color scheme?

Reply:
No it isn't, I have changed it to reflect the colour palette now. Thanks